packages/expo-screen-orientation/src/ScreenOrientation.types.ts

```diff
@@ -1,31 +1,22 @@
 export enum Orientation {
-  UNKNOWN = 'UNKNOWN',
-  PORTRAIT = 'PORTRAIT',
-  PORTRAIT_UP = 'PORTRAIT_UP',
-  PORTRAIT_DOWN = 'PORTRAIT_DOWN',
-  LANDSCAPE = 'LANDSCAPE',
-  LANDSCAPE_LEFT = 'LANDSCAPE_LEFT',
-  LANDSCAPE_RIGHT = 'LANDSCAPE_RIGHT',
+  UNKNOWN = 0,
+  PORTRAIT_UP = 1,
+  PORTRAIT_DOWN = 2,
+  LANDSCAPE_LEFT = 3,
+  LANDSCAPE_RIGHT = 4,
 }
 
 export enum OrientationLock {
-  DEFAULT = 'DEFAULT',
-  ALL = 'ALL',
-  PORTRAIT = 'PORTRAIT',
-  PORTRAIT_UP = 'PORTRAIT_UP',
-  PORTRAIT_DOWN = 'PORTRAIT_DOWN',
-  LANDSCAPE = 'LANDSCAPE',
-  LANDSCAPE_LEFT = 'LANDSCAPE_LEFT',
-  LANDSCAPE_RIGHT = 'LANDSCAPE_RIGHT',
-  OTHER = 'OTHER',
-  UNKNOWN = 'UNKNOWN',
-  ALL_BUT_UPSIDE_DOWN = 'ALL_BUT_UPSIDE_DOWN', // deprecated
-}
-
-export enum SizeClassIOS {
-  REGULAR = 'REGULAR',
-  COMPACT = 'COMPACT',
-  UNKNOWN = 'UNKNOWN',
+  DEFAULT = 0,
+  ALL = 1,
+  PORTRAIT = 2,
+  PORTRAIT_UP = 3,
+  PORTRAIT_DOWN = 4,
+  LANDSCAPE = 5,
+  LANDSCAPE_LEFT = 6,
+  LANDSCAPE_RIGHT = 7,
+  OTHER = 8,
```

Comment:
I can't imagine how `OTHER` would be different than `UNKNOWN`. Would it make sense to unify them to one `UNKNOWN` or is there some scenario where the difference makes sense?